lib/tasks/deployment/20230729145858_convert_learning_hour_types.rake

```diff
@@ -0,0 +1,25 @@
+namespace :after_party do
+  desc "Deployment task: converts_learning_type_enum_to_learning_hour_types_for_casa_orgs"
+  task convert_learning_hour_types: :environment do
+    puts "Running deploy task 'convert_learning_hour_types'"
+
+    learning_types = %w[book movie webinar conference other]
+    CasaOrg.all.each do |casa_org|
+      user_ids = casa_org.users.pluck(:id)
+      learning_hours = LearningHour.where(user_id: user_ids)
+
+      learning_types.each do |learning_type|
+        learning_hour_type = casa_org.learning_hour_types.new(name: learning_type.capitalize)
+        learning_hour_type.position = 99 if learning_type == "other"
+        learning_hour_type.save
```

Comment:
Can we do a `find_or_create_by!` or similar with something that will noisily fail in case there are any issues?